spec/features/govuk_verify_spec.rb

```diff
@@ -100,4 +100,20 @@
       expect(@claim.verify_response).to eq(parsed_vsp_translated_response("authentication-failed"))
     end
   end
+
+  context "user takes a particularly long time to get through verify" do
+    before do
+      stub_vsp_translate_response_request
+    end
+
+    scenario "Does not time the user out", js: true do
```

Comment:
can we have another spec (or modify this existing one?) to show that it still times out after the extended timeout period?